addon/webextension/background/senderror.js

```diff
@@ -92,20 +102,26 @@ this.senderror = (function() {
       return;
     }
     if (!Raven.isSetup()) {
-      Raven.config(dsn, {allowSecretKey: true}).install();
+      Raven.config(dsn, {allowSecretKey: true, dataCallback: normalizeURL}).install();
     }
-    let exception = new Error(e.message);
+    // NOTE: don't use e.message, as that is a localized string
+    let exception = new Error(e.popupMessage);
     exception.stack = e.multilineStack || e.stack || undefined;
+
+    // To improve Sentry grouping, replace the UUID in the moz-extension URL with the string 'screenshots-addon'.
+    // To improve error reporting, replace the moz-extension scheme with resource: raven.js doesn't parse errors at moz-extension URLs.
+    exception.stack = exception.stack.replace(/moz-extension:\/\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/g, "resource://screenshots-addon")
     let rest = {};
     for (let attr in e) {
       if (!["name", "message", "stack", "multilineStack", "popupMessage", "version", "sentryPublicDSN", "help"].includes(attr)) {
         rest[attr] = e[attr];
       }
     }
-    rest.stack = e.multilineStack || e.stack;
+    rest.stack = exception.stack;
     Raven.captureException(exception, {
       logger: 'addon',
       tags: {version: manifest.version, category: e.popupMessage},
+      culprit: e.popupMessage,
```

Comment:
oddly the 'culprit' is always set to moz-extension.../_generated_background_page if we don't manually set this